lib/aws.js

```diff
@@ -10,15 +10,17 @@ const logger = require('./logger');
 const config = require('./config');
 
 module.exports.init = function(callback) {
-    // AWS will look relative to the Node working directory, not the current
```

Comment:
The `lib/workspace` version used `process.cwd()`, so I ported that here and removed this comment. Should this be switched back to `./aws-config.json`?

Reply:
I'm honestly not really sure about when it would matter either way. I guess it's fine as is? But I don't have a strong preference.